app/queries/base_path_state.rb

```diff
@@ -0,0 +1,71 @@
+module Queries
+  module BasePathState
+    extend ArelHelpers
+
+    def self.conflict_with_draft_content(content_item_id, base_path)
+      content_items_table = table(:content_items)
+      states_table = table(:states)
+      locations_table = table(:locations)
+      translations_table = table(:translations)
+
+      scope = content_items_table
+        .project(
+          content_items_table[:id],
+          content_items_table[:content_id],
+          translations_table[:locale]
+        )
+        .join(states_table).on(
+          content_items_table[:id].eq(states_table[:content_item_id])
+        )
+        .join(locations_table).on(
+          content_items_table[:id].eq(locations_table[:content_item_id])
+        )
+        .outer_join(translations_table).on(
+          content_items_table[:id].eq(translations_table[:content_item_id])
+        )
+        .where(content_items_table[:id].not_eq(content_item_id))
+        .where(states_table[:name].eq("draft"))
+        .where(locations_table[:base_path].eq(base_path))
+        .order(content_items_table[:created_at].desc)
+        .take(1)
+      get_rows(scope).first.try(:symbolize_keys)
+    end
+
+    def self.conflict_with_live_content(content_item_id, base_path)
```

Comment:
This doesn't seem to match what's in the commit message:

> - A live content item fails validation if there is another live content
  item with the same content id